app/views/Project/Tagging/Sources/SourcesStats/index.tsx

```diff
@@ -111,18 +101,18 @@ function SourcesStats(props: Props) {
                         <IoDocument />
                     )}
                     label={_ts('sourcesStats', 'totalEntries')}
-                    value={totalEntries}
+                    value={numberOfEntries ?? 0}
                     variant="accent"
                 />
                 <ProgressLine
-                    progress={(totalVerifiedEntries / totalEntries) * 100}
-                    title={_ts('sourcesStats', 'totalVerifiedEntries')}
+                    progress={((totalVerifiedEntries ?? 0) / (numberOfEntries ?? 0)) * 100}
+                    title="Entries Verified"
                     variant="complement1"
                     size="large"
                 />
                 <ProgressLine
-                    progress={(totalUnverifiedEntries / totalEntries) * 100}
-                    title={_ts('sourcesStats', 'totalUnverifiedEntries')}
+                    progress={((totalControlledEntries ?? 0) / (numberOfEntries ?? 0)) * 100}
```

Comment:
We should prevent division by zero.
Let's create a small helper to calculate percentages or check if we have one in fujs